<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatoiRFileWatcherClass</class>
 <widget class="QMainWindow" name="MatoiRFileWatcherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MatoiRFileWatcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MatoiRFileWatcher.qrc">
    <normaloff>:/MatoiRFileWatcher/res/watch.png</normaloff>:/MatoiRFileWatcher/res/watch.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="watcherList">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="watcherPath">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="browsePath">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MatoiRFileWatcher.qrc"/>
 </resources>
 <connections/>
</ui>
